app/src/main/java/com/infomaniak/swisstransfer/ui/screen/newtransfer/ImportFilesViewModel.kt

```diff
@@ -185,6 +188,12 @@ class ImportFilesViewModel @Inject constructor(
         uploadManager.removeAllUploadSession()
     }
 
+    private suspend fun retrieveLastAuthorEmail() {
+        appSettingsManager.appSettings.first()?.let { appSettings ->
+            appSettings.lastAuthorEmail?.let { lastAuthorEmail -> transferAuthorEmail = lastAuthorEmail }
```

Comment:
Leaving `it` here would be less repetitive, and therefore, easier to read.

Reply:
Agreed